events/lifestyles/warfare_lifestyle/martial_strategy_events.txt

```diff
@@ -363,6 +363,38 @@ martial_strategy.1051 = {
 		add_prestige = medium_prestige_gain
 		add_dread = minor_dread_gain
 	}
+	option = {
+		name = martial_strategy.1051.g.wc
+		trigger = {
+			AND = {
+				has_trait = sadistic
+				has_trait = arbitrary
+			}
+		}
+		trait = sadistic
+		trait = arbitrary
+
+		set_nickname_effect = { NICKNAME = nick_the_ice_hearted }
+		add_dread = major_dread_gain
+		add_prestige = minor_prestige_gain
+	}
+
+	option = {
+		name = martial_strategy.1051.h.wc
+		trigger = {
+			AND = {
+				has_trait = ambitious
+				has_trait = strong
+				martial >= 15
+			}
+		}
+		trait = ambitious
+		trait = strong
```

Comment:
Strong is rare educational trait. To make it more common.
```suggestion
			has_trait = ambitious
			prowess > medium_skill_rating
		}
		trait = ambitious
```

Reply:
done in 4cbd2fc7